### ISSUE TITLE
Build 'Header': Overhaul the existing Navigation bar

### ISSUE THREAD

**Issue by levsthings** [hollowverse-archive/hollowverse#64]
I'm working on a complete overhaul of the Navbar which will be soon contained in `Header.tsx`. I need some clarification on some the things that we want in the Navigation Bar for v1 so that I can translate the needed functionality to a universally accepted UI language for mobile devices. 

Please comment on the functionality of the Navigation bar listed below:

1. "Hollowverse" text which links to the homepage.  - **Makes it to v1**
2. Login/Logout - **Makes it to v1.** Do we plan on implementing any other authentication methods other than Facebook in the foreseeable future?
3. Search - **Does it make it to v1?**
4. Any other functionality that I haven't mentioned. - **Do they make it to v1?**

Basically, with these clarified, I'll figure out a way to present these elements and understand if we need a Hamburger menu or not.

**Comment by wholesomedev** [comment]
> 1. "Hollowverse" text which links to the homepage.  - **Makes it to v1**

👍 

> 2. Login/Logout - **Makes it to v1.** Do we plan on implementing any other authentication > methods other than Facebook in the foreseeable future?

Nope, just Facebook for the foreseeable future.

> 3. Search - **Does it make it to v1?**

Yes. As laid out in the [Product Description](https://github.com/hollowverse/hollowverse/wiki/Product-Description#homepage), search will be on the homepage. If you plan to add a search icon to the header, perhaps all it would have to do is direct to the homepage and focus the search box. (Even though that's not a good user experience, it's good enough for v1. Other UI/UX may get complicated.)

> 4. Any other functionality that I haven't mentioned. - **Do they make it to v1?**

v1 scope isn't definitively set. More features may creep in. But the idea is to have only a minimal viable product and minimal essential infrastructure for v1.
